components/test/src/main/java/org/trellisldp/test/LdpBasicContainerTests.java

```diff
@@ -289,7 +289,8 @@ default void testCreateContainerViaPut() {
             final Graph g = readEntityAsGraph(res.getEntity(), getBaseURL(), TURTLE);
             final IRI identifier = rdf.createIRI(getContainerLocation());
             assertAll("Check the resulting graph", checkRdfGraph(g, identifier));
-            assertTrue(g.contains(identifier, LDP.contains, rdf.createIRI(child4)), "Check for an ldp:contains triple");
+            assertFalse(g.contains(identifier, LDP.contains, rdf.createIRI(child4)),
```

Comment:
Just as a side note, this test is failing for me now, I think it will pass after this PR, yay!